docs/best-practices/continuous-integration.md

```diff
@@ -492,6 +492,16 @@ In case you wish to run tests as part of the main build step, we suggest that yo
 trainer(output_directory: ENV["NEVERCODE_XUNIT_RESULTS_DIR"])
 ```
 
+Nevercode detects your **_fastlane_ tests for Android** automatically. However, tests are disabled by default and have to be enabled manually.
+
+1. In your app settings, navigate to the **Test** tab.
+1. Click on **_fastlane_ Gradle** to expand it.
+1. Select **Enabled** and type the **lane name** for tests as specified in your `Fastfile`.
+1. If you would like to run the tests on Android emulator, make sure to select **Require Android emulator**.
+1. Click **Save** to finish the setup. Your _fastlane_ tests will be enabled and run each time you build your app.
+
+You can also use **AWS Device Farm** for testing.
```

Comment:
Only on Android or for both? If for both, this should probably be moved to an introductory paragraph after "Run tests" that explains how tests are normally run and that AWS is an alternative.